module/VuFind/src/VuFind/AjaxHandler/AjaxHandlerInterface.php

```diff
@@ -44,6 +44,7 @@ interface AjaxHandlerInterface
     const STATUS_HTTP_BAD_REQUEST = 400; // bad request
     const STATUS_HTTP_NEED_AUTH = 401;   // must login first
     const STATUS_HTTP_FORBIDDEN = 403;   // method is unavailable
+    const STATUS_HTTP_NOT_FOUND = 404;   // object/data not found
```

Comment:
This new constant does not appear to be used anywhere; I don't object to adding it, but just curious if something is missing.